pkg/wsman/cim/chip/types.go

```diff
@@ -41,13 +40,44 @@ type (
 	}
 
 	PackageResponse struct {
-		XMLName           xml.Name                 `xml:"CIM_Chip"`
-		CanBeFRUed        bool                     `xml:"CanBeFRUed"`
-		CreationClassName string                   `xml:"CreationClassName"`
-		ElementName       string                   `xml:"ElementName"`
-		Manufacturer      string                   `xml:"Manufacturer"`
-		OperationalStatus models.OperationalStatus `xml:"OperationalStatus"`
-		Tag               string                   `xml:"Tag"`
-		Version           string                   `xml:"Version"`
+		XMLName           xml.Name          `xml:"CIM_Chip"`
+		CanBeFRUed        bool              `xml:"CanBeFRUed"`        // Boolean that indicates whether this PhysicalElement can be FRUed (TRUE) or not (FALSE).
+		CreationClassName string            `xml:"CreationClassName"` // CreationClassName indicates the name of the class or the subclass used in the creation of an instance. When used with the other key properties of this class, this property allows all instances of this class and its subclasses to be uniquely identified.
+		ElementName       string            `xml:"ElementName"`
+		Manufacturer      string            `xml:"Manufacturer"` // The name of the organization responsible for producing the PhysicalElement. This organization might be the entity from whom the Element is purchased, but this is not necessarily true. The latter information is contained in the Vendor property of CIM_Product.
+		OperationalStatus OperationalStatus `xml:"OperationalStatus"`
```

Comment:
OperationalStatus should have "// Indicates the current statuses of the element." as a comment.